### ISSUE TITLE
[Feature]: Improve text logic for Sensory words

### ISSUE THREAD

**Issue by mbgower** [IBMa/equal-access#775]
### Project

rule-server

### Description

From Slack:
>Can someone explain why stop is flagged when used in documentation by the accessibility checker? Also, can we request better intelligence for other words that get flagged. Cases of concern:
stop as part of a command (e.g., daylight-stop-minute)
top when top-level and is about hierarchy
square in square root
left and right when discussing sequence and its reversal (e.g., left-to-right and right-to-left)
These cases all show that a simple parse leads to more validation by writer BUT with intelligence (parsing with preceding and following "word") can more accurately identify where an accessibility concern might exist.

**Comment by tombrunet** [comment]
https://codesandbox.io/s/issue-775-eky9qt?file=/index.html
@mbgower I agree with the first three. left-to-right still feels questionable to me... it somewhat depends on what they're describing should be left-to-right, does it not?

**Comment by mbgower** [comment]
Yeah, I agree that that is the least clearly not related to sensory characteristics in these. I'd be inclined to tackle the ones you've IDed and leave left and right in place. I assume you already have some logic in to not flag every occurrence of "right"?

**Comment by shunguoy** [comment]
we need to use regex to find the sensory words. use a list of sensory words to do.

**Comment by shunguoy** [comment]
look at the exception list of words as well. make sure the help content.

**Comment by shunguoy** [comment]
we have a rule for this, but need to enhance the rule

**Comment by MHoov** [comment]
Was this addressed by #1641 @tombrunet
